CondFormats/DataRecord/src/ES_Darkening.cc

```diff
@@ -0,0 +1,4 @@
+#include "FWCore/Utilities/interface/typelookup.h"
+#include "CondFormats/HcalObjects/interface/HBHEDarkening.h"
+
+TYPELOOKUP_DATA_REG(HBHEDarkening);
```

Comment:
This is a casualty of multiple refactorings - I will rename it to use the standard form in `CondFormats/HcalObjects`